libs/dal/src/repositories/message/message.repository.ts

```diff
@@ -245,6 +245,10 @@ export class MessageRepository extends BaseRepository<MessageDBModel, MessageEnt
     await this.message.delete({ _id: message._id, _environmentId: message._environmentId });
   }
 
+  async deleteMany(query: MessageQuery) {
+    return await this.message.delete({ ...query, deleted: false });
```

Comment:
not sure if `deleted: false` is needed here.

Reply:
It will delete only those values which are not yet deleted